common/persistence/tests/history_store.go

```diff
@@ -113,7 +113,7 @@ func (s *HistoryEventsSuite) TestAppendSelect_First() {
 	shardID := rand.Int31()
 	treeID := uuid.New()
 	branchID := uuid.New()
-	branchToken, err := p.NewHistoryBranchToken(treeID, branchID, []*persistencespb.HistoryBranchRange{})
+	branchToken, err := p.CreateHistoryBranchToken(treeID, branchID, []*persistencespb.HistoryBranchRange{})
```

Comment:
I think the branch token should be constructed by calling persistence interface, instead of calling the underlying impl directly.